styles/layout.less

```diff
@@ -352,6 +352,30 @@ footer, #footer {
     }
 }
 
+/* ------------------------------------------------------------------------ */
+/* Disclosure boxes with optional info */
+
+details {
+    border: 1px solid #aaa;
```

Comment:
We should try not to hard-code color in `layout.less` because it may not work for all themes. See if one of the pre-defined `@border-color-base*` values will work for what you want as the themes override these.

Reply:
ACK. Yay for CSS variables.